.github/workflows/continuous_integration.yml

```diff
@@ -0,0 +1,46 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - develop
+      - main
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  build:
+    name: build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout commit
+        uses: actions/checkout@v2
+
+      - name: Install correct version of Node
+        shell: bash -l {0}
+        run: nvm install
```

Comment:
Turns out the default Ubuntu runner includes `nvm`! You need to set up a login shell for it to work, but it takes for the rest of the job. Takes 2 - 3 seconds to install the correct version.

Reply:
For the context of CI, it seems more important to be able to support execution against a range of Node versions than the one the project is specifically pinned to. That makes me more inclined to pursue a `setup-node` Action approach with a pin on version 12.x to start.

What do you think?

Reply:
Good point, this is a library and not just an application. I think pinning the publish to a static version makes sense from the perspective of reproducible builds, but tests should be run against multiple supported versions. I think active LTS versions are a good starting point.

Reply:
After thinking about this, I feel less strongly about pinning the publish process than I originally thought. We can easily see which versions were used in the build in case we need to reproduce failure, and we're exercising the version we will use to publish in the CI workflow. I'm going to change the release workflow to use `setup_node` and whichever version of Yarn comes with it as well. `.nvmrc` and `.yvmrc` can just be for development convenience and consistency.